Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -204,17 +185,47 @@ function prepareMergeConcept(conceptId) {
         conceptsToMerge = conceptsToMerge + "," + conceptId;    
     }
 }
-
 function createWrapper(wrapperId) {
   window.location = '${pageContext.servletContext.contextPath}/auth/conceptlist/addconceptwrapper?wrapperId=' + wrapperId;
 }
-
+var conceptId;
+ 
+	$(document).ready(function(){
+	$(".btnReview").click(function() {
+		conceptId= $(this).closest("tr")   // Finds the closest row <tr> 
+	                       .find(".entryID")     // Gets a descendent with class="entryID"
+	                       .text();         // Retrieves the text within <td>
+
+   $("#conceptId").val(conceptId);	      // Outputs the answer
+		});
+	});
+var rowNum;  
+function  getId(element) {
+	rowNum =  element.rowIndex;
+}
+var formDisplayed=true;
```

Comment:
what are you trying to do with this? jquery modals can simply be shown or hidden, non global variables needed.

Reply:
?